logging/simplelog/simplelog.go

```diff
@@ -0,0 +1,134 @@
+// iample simplelog writes some entries, lists them, then deletes the log.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	// [START imports]
+	// NOTE: This will become cloud.google.com/go/logging soon.
+	"cloud.google.com/go/preview/logging"
+
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+	// [END imports]
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Usage: simplelog <project-id>")
+		os.Exit(2)
+	}
+	projID := os.Args[1]
+
+	// [START setup]
+	ctx := context.Background()
+	client, err := logging.NewClient(ctx, projID,
+		// Admin scope is required to delete logs.
+		option.WithScopes(logging.AdminScope))
+
+	if err != nil {
+		log.Fatalf("Failed to create logging client: %v", err)
+	}
+	client.OnError = func(err error) {
+		// Print an error to the local log.
+		// For example, if Flush() failed.
+		log.Printf("client.OnError: %v", err)
+	}
+	// [END setup]
+
+	log.Print("Writing some log entries.")
+	writeEntry(client)
+	structuredWrite(client)
+
+	// It takes some time for the log entries to appear.
```

Comment:
Sure - either way. I'm neurotic about sleeps in code, but it's fine if you're cool with. :)